lib/fizzy/parser.cpp

```diff
@@ -64,6 +64,12 @@ parser_result<ValType> parse(const uint8_t* pos, const uint8_t* end)
     }
 }
 
+const uint8_t* validate_valtype(const uint8_t* pos, const uint8_t* end)
+{
+    const auto [_, next_pos] = parse<ValType>(pos, end);
```

Comment:
Why is this better than `std::tie(std::ignore, pos) = parse...` ?

Reply:
Can be, but it will weirdly update the input argument.